ec2api/api/route_table.py

```diff
@@ -480,18 +480,26 @@ def _update_routes_in_associated_subnets(context, route_table, cleaner,
 
 def _update_subnet_host_routes(context, subnet, route_table, cleaner=None,
                                rollback_route_table_object=None,
-                               router_objects=None, neutron=None):
+                               router_objects=None, neutron=None,
+                               vpc_route_revert=False, add_vpc_route=False):
     neutron = neutron or clients.neutron(context)
     os_subnet = neutron.show_subnet(subnet['os_id'])['subnet']
     gateway_ip = str(netaddr.IPAddress(
         netaddr.IPNetwork(os_subnet['cidr']).first + 1))
-    host_routes = _get_subnet_host_routes(context, route_table, gateway_ip,
-                                          router_objects)
+    if add_vpc_route == False:
+        host_routes = _get_subnet_host_routes(context, route_table, gateway_ip,
+                                              router_objects)
+    else:
+        host_routes = _get_subnet_host_routes(context, route_table, gateway_ip,
+                                              router_objects, True)
     neutron.update_subnet(subnet['os_id'],
                           {'subnet': {'host_routes': host_routes}})
     if cleaner and rollback_route_table_object:
         cleaner.addCleanup(_update_subnet_host_routes, context, subnet,
                            rollback_route_table_object)
+    elif vpc_route_revert == True:
```

Comment:
in subnet.py#L275
If you can change it like below statement, then it will be more readable.
route_table_api._update_subnet_host_routes(context, subnet, main_route_table, cleaner, rollback_route_table_object=None, router_objects=None, neutron=None,
                               vpc_route_revert=False)